### PR TITLE
fix(pt-br): remove duplicate headings

### PR DESCRIPTION
**Pull request by yin1999**
### Description

remove duplicate headings. Since I don't speak Portuguese, please feel free to correct my mistakes.

### Motivation

Prepare for enabling [the `no-duplicate-heading` markdown-lint rule](https://github.com/mdn/translated-content/blob/d13c47b2b819b24e7623bb55ad5f24606c537d87/.markdownlint.jsonc#L20-L22).
Part of #17582

### CODE REVIEW

File: files/pt-br/web/api/filereader/index.md [hunk 1]
```diff
@@ -83,76 +83,3 @@ Ouça esses eventos usando {{domxref("EventTarget/addEventListener", "addEventLi
 - {{domxref("File")}}
 - {{domxref("Blob")}}
 - {{domxref("FileReaderSync")}}{ APIRef("File API")}}
-
```
Comment (yin1999) [inline files/pt-br/web/api/filereader/index.md:86]: The following content is duplicated

File: files/pt-br/web/api/xmlhttprequest/index.md [hunk 2]
```diff
@@ -17,22 +17,6 @@ var myRequest = new XMLHttpRequest();
 
 Para obter detalhes sobre como usar XMLHttpRequest , consulte [Usando XMLHttpRequest](/pt-BR/DOM/XMLHttpRequest/Using_XMLHttpRequest).
 
-## Métodos
```
Comment (yin1999) [inline files/pt-br/web/api/xmlhttprequest/index.md:20]: We no longer document WebIDL, and those methods have been documented below.

File: files/pt-br/web/html/element/input/range/index.md [hunk 3]
```diff
@@ -235,42 +235,45 @@ Você pode adicionar rótulos para seu controle usando o atributo [`label`](/pt-
...
+input[type="range"] {
+  appearance: slider-vertical;
+}
+```
 
-{{EmbedLiveSample("Orientation_sample2", 200, 200, "orientation_sample2.png")}}
+{{EmbedLiveSample("Usando a propriedade appearance", 200, 200)}}
 
-**Atualmente, nenhum dos principais navegadores suporta a criação de entradas de alcance vertical usando o CSS desta maneira, mesmo que seja a maneira como a especificação recomenda que eles o façam.**
+#### Controle de range vertical
```
Comment (yin1999) [inline files/pt-br/web/html/element/input/range/index.md:272]: The following content has not been updated (we can do this in the future)

File: files/pt-br/web/javascript/reference/global_objects/string/charcodeat/index.md [hunk 4]
```diff
@@ -77,41 +77,6 @@ function fixedCharCodeAt(str, idx) {
 }
 ```
 
-### Corrigindo o `charCodeAt()` para manipular caracteres de Plano Multilingual não Básico se sua presença na string é desconhecida
```
Comment (yin1999) [inline files/pt-br/web/javascript/reference/global_objects/string/charcodeat/index.md:80]: Duplicate heading, and the following content will be removed from the English document

**Review by josielrocha — APPROVED** [review]

**Comment by josielrocha** [comment]
Thank you for your help @yin1999!

**Pull request merged**